runtime/service/src/main/java/org/apache/polaris/service/admin/ApiBusinessLogic.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.polaris.service.admin;
+
+import jakarta.inject.Qualifier;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Qualifier
+@Retention(RUNTIME)
+@Target({TYPE, METHOD, FIELD, PARAMETER})
+public @interface ApiBusinessLogic {
```

Comment:
I'm again the term "business logic". I find it very confusing. How about `PublicApi`?

Reply:
Side note: this interface appears to be out of scope based on the PR description.

Reply:
I see your idea now, I think :sweat_smile: 

However, I'm not sure this is the best approach. Current delegators tae concrete classes into their constructors - this reduces option for composing the delegators with other implementations.

Since delegation is apparently covering API interfaces, I believe it is best to have one delegator per `*ApiService` interface.

Then, I propose to use `@MainService` as a `@Qualifier` for `PolarisServiceImpl`, etc.

For completeness, we should probably use a qualifier like `@EventsService` for delegators that introduce events.

This way, both sets of services can be disambiguated in this repo and in downstream projects.